compatibilitytool.vdf

```diff
@@ -0,0 +1,13 @@
+"compatibilitytools"
+{
+  "compat_tools"
+  {
+    "proton-ge-custom_flatpak"
+    {
+      "display_name" "Proton GE Custom @VERSION@ (community build @BUILD@)"
```

Comment:
I think "community" doesn't make much sense here, since Proton-GE itself isn't an "official" thing. Ping @nanonyme

Reply:
Well, it's a community build of a community Proton distribution :smile:

Reply:
Also, won't the name be too long? Is the version still visible in Steam UI?

Reply:
Yep.

![Capture d’écran de 2020-12-01 15-02-53](https://user-images.githubusercontent.com/5477973/100750510-6da5a900-33e6-11eb-9c10-be599cbe8d74.png)